### PR TITLE
270457 Sector: Maternal and Newborn Health

### PR DESCRIPTION
**Pull request by mkaczmarczyksoldevelo**
@biyeun I have not found this [http://sites.dimagi.com/case-study/mhealth-care-motech?__hstc=240960668.09d92d17ff9643d1120d820f64ccaf6f.1475454337459.1475454337459.1475454337459.1&__hssc=240960668.3.1475454337459&__hsfp=3317662926](case study) in the new dimagi.com repo. I've added this to dimagi/case_studies/data/mhealth/mHealth_care_motech.py but I dont have the dropbox URL to the CS pdf file. Could you provide one? Or correct me if there is proper CS page for this one.

### CODE REVIEW

**Comment by biyeun** [comment]
Great! left a few comments.

File: dimagi/case_studies/data/mhealth/__init__.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     supply_chain_tanzania,
     tdh_burkinafaso,
     world_vision_motech,
+    mHealth_care_motech,
```
[the remark this answers is not in the document]
Comment (mkaczmarczyksoldevelo) [reply dimagi/case_studies/data/mhealth/__init__.py]: Done

File: dimagi/case_studies/data/mhealth/mHealth_care_motech.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        ugettext_lazy("Mobile reports"),
+        ugettext_lazy("Home visit scheduler"),
+        ugettext_lazy("Call center"),
+        ugettext_lazy("Referral system"),
+        ugettext_lazy("EDD & BMI calculators"),
+        ugettext_lazy("Multimedia"),
+        ugettext_lazy("Performance dashboards"),
+    ],
+    slug="mHealth-care-motech",
+    download_url=None,
```
[the remark this answers is not in the document]
Comment (mkaczmarczyksoldevelo) [reply dimagi/case_studies/data/mhealth/mHealth_care_motech.py]: Done

**Comment by mkaczmarczyksoldevelo** [comment]
@biyeun I've noticed that we lack the cover image for the Motech case study. Do you have a script that converts the first page of the pdf file to a jpg (like Image Magic library)? Or you are using Gimp/Photoshop?

**Pull request merged**